### PR TITLE
replaced maven ignore with lifecycle mapping to ensure smooth import …

### PR DESCRIPTION
**Pull request by martinlippert**
…into STS/Eclipse

### CODE REVIEW

**Comment by martinlippert** [comment]
would be great if this could be pulled into the main repo soon so allow a smoother experience using the latest STS builds (and the upcoming STS 3.7.1.RC1 version)

File: spring-boot-tools/spring-boot-gradle-plugin/pom.xml [hunk 1]
```diff
@@ -84,22 +84,21 @@
...
-										</groupId>
-										<artifactId>
-											maven-compiler-plugin
-										</artifactId>
-										<versionRange>
-											[2.10,)
-										</versionRange>
+										<groupId>org.apache.maven.plugins</groupId>
+										<artifactId>maven-compiler-plugin</artifactId>
+										<versionRange>[2.0,)</versionRange>
```
Comment (wilkinsona) [inline spring-boot-tools/spring-boot-gradle-plugin/pom.xml]: What's the reason for changing the minimum version from 2.10 to 2.0?

Comment (martinlippert) [inline spring-boot-tools/spring-boot-gradle-plugin/pom.xml]: No specific reason, must be an accident. Should I update the pull request or would you like to change that on your end?

Comment (martinlippert) [inline spring-boot-tools/spring-boot-gradle-plugin/pom.xml]: (and thanks for pointing this out)

Comment (wilkinsona) [inline spring-boot-tools/spring-boot-gradle-plugin/pom.xml]: Thanks. I'll take care of it during the merge.

Comment (martinlippert) [inline spring-boot-tools/spring-boot-gradle-plugin/pom.xml]: Great, thanks! Much appreciated!

**Pull request closed without merging**